app/web/components/AppRoute.tsx

```diff
@@ -57,6 +59,7 @@ export const useAppRouteStyles = makeStyles((theme) => ({
   },
   "@global body": {
     height: "100%",
+    overflow: "hidden",
```

Comment:
The whole app seems to be unscrollable apart from the message pages, and even the chat box is buggy where it's completely hidden for me 🤔

https://github.com/user-attachments/assets/1090d51a-a721-44a8-bdd1-893d122e3f7a

(Excuse the weird light in the top right lol, I need a better screen reader app that doesn't show my room ceiling 😆)

Reply:
It seems to break scrolling even on desktop (no scrollbar for home page)
![Screenshot 2024-10-28 at 01 04 35](https://github.com/user-attachments/assets/cfbeb8f7-8f52-4153-ad53-59003a6ff961)

Reply:
Hmmm I guess that was a bit ambitious of me to sneak into this PR...I'll take it out because you're right it probably needs a separate thorough testing.

Reply:
Just made a separate ticket #5070 . That's probably a good first issue for one of the new people as well!